### ISSUE TITLE
Link titles not supported?

### ISSUE THREAD

**Issue by DamianMehers** [apple/swift-markdown#67]
[Example 481](https://spec.commonmark.org/0.30/#example-481 "title goes here") of the commonmark spec shows the use of a title in a link:

```markdown
[link](/uri "title")
```

however I don't see the title property in the [Link](https://apple.github.io/swift-markdown/documentation/markdown/link) documentation

When I test, it seems as though the title is discarded:
```swift
let document = Document(parsing: "[link](/uri \"title\")")
print(document.debugDescription())
```

Outputs:
```
Document
└─ Paragraph
   └─ Link destination: "/uri"
      └─ Text "link"
```

Title is supported for Images:
```swift
let document2 = Document(parsing: "![foo](/url \"title\")")
print(document2.debugDescription())
```

Outputs the title:
```
Document
└─ Paragraph
   └─ Image source: "/url" title: "title"
      └─ Text "foo"
```

Is this a version thing?

Thanks,
   Damian

**Comment by QuietMisdreavus** [comment]
I think this is a bug in swift-markdown. It looks like it doesn't read the link title from cmark:

https://github.com/apple/swift-markdown/blob/87c9e60b73174643f1f0bdfd81d41ce379defde0/Sources/Markdown/Parser/CommonMarkConverter.swift#L445-L454

There are a couple other places where this happens (table column alignments and [ordered-list starting indices](https://github.com/apple/swift-markdown/pull/22), for example), so this is probably an oversight from the original implementation.

**Comment by LiYanan2004** [comment]
In my opinion, The child of the Link is the label. In this case, 'foo' is the Label

**Comment by jurex** [comment]
hey guys, any progress with this issue? Would love to see it fixed. Thanks!

**Comment by Kyle-Ye** [comment]
> hey guys, any progress with this issue? Would love to see it fixed. Thanks!

I've opened a potential fix for it. If someone can help review it, it can be merged into main soon and might be cherry-picked to land on release/5.9.

**Comment by Kyle-Ye** [comment]
Hello @DamianMehers, this feature is merged in main today. You may have a try on it and any feedback is greatly appreciated. And I believe we can have it landed on the release of Swift 5.10.
